lsp.opam

```diff
@@ -26,7 +26,8 @@ depends: [
   "dune" {>= "3.0"}
   "jsonrpc" {= version}
   "yojson"
-  "ppx_yojson_conv_lib" {>= "v0.14"}
+  "ppx_yojson_conv" {>= "v0.14"}
```

Comment:
Do we need the preprocessor? We're only using the runtime

Reply:
@rgrinberg Oops, I guess we actually needed different changes. Updated PR.